types/discord.ts

```diff
@@ -924,8 +924,7 @@ export interface DiscordActivitySecrets {
   match?: string;
 }
 
-// https://github.com/discord/discord-api-docs/pull/2219
-// TODO: add documentation link
+/** https://discord.com/developers/docs/topics/gateway#activity-object-activity-buttons */
```

Comment:
Tell me if I should move this to another PR

Reply:
I think this is the answer 🙃